### ISSUE TITLE
✨  Improve icons

### ISSUE THREAD

**Issue by liaprins-czi** [napari/napari#3195]
### Overview of design need
- **Is there an existing GitHub issue this design work pertains to?** 
[Use font icons instead of SVGs #3136](https://github.com/napari/napari/issues/3136)
[Heuristic: Iconography #16](https://github.com/napari/product-heuristics-2020/issues/16)

- **In a sentence or two, describe the problem to be solved for users**
Overhaul the design of all the icons in the viewer to be more communicative of the function they represent and more cohesive as a set. If it makes sense from a design perspective to use existing font libraries (e.g. [Font Awesome](https://fontawesome.com/v5.15/icons?d=gallery&p=2&m=free), [Material UI icons](https://fonts.google.com/icons)), this will make the implementation easier (these likely won't have the specificity / custom meanings we need, though). If not using existing fonts, then try to have [all icons' final designs ready at once](https://github.com/napari/napari/issues/3136#issuecomment-900669688) to simplify implementation.

### What level of design is needed? (Choose all that apply)
_This section may be updated by the designer / UX researcher working on this issue_

- [x] _Maybe_ **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [ ] **information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [x] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [X] **Yes:** engineering cannot proceed without a design first
- [ ] **No:** engineering can create a first version, and design can come in later to iterate and refine

- **If selecting _Yes_, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?**
No need for engineering to do anything until final icons are ready.

### T-shirt size
- [ ] ●○○○○ XS: able to be done in a typical workday or two
- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
- [ ] ●●●○○ M: able to do within one sprint, if it is the sole focus
- [X] ●●●●○ L: will likely take more than one sprint to complete, but less than two
- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further

**Comment by tlambert03** [comment]
sounds great!
Want to just point out react icons, which has a useful page for searching a lot of font families on the same pages:
https://react-icons.github.io/react-icons
and icomoon https://icomoon.io/app/#/select

**Comment by liaprins-czi** [comment]
Awesome, thanks @tlambert03 !

**Comment by isabela-pf** [comment]
I noticed that the above resources aren't including the extensive community effort at [Material Design Icons](https://materialdesignicons.com/) (a fork of Google's Material Icons with a lot of icons added). I know some other projects have had luck finding icons that fulfill their needs there. Here is the [full list of icons for the latest version](https://pictogrammers.github.io/@mdi/font/6.1.95/).

Hope that helps!

**Comment by tlambert03** [comment]
yep, those are amazing!  ~(they're included in the massive collection in the react-icons viewer [here](https://react-icons.github.io/react-icons/icons?name=md)) ...~

**Comment by tlambert03** [comment]
sorry!  didn't mean to close this :joy: ... what I meant to say is that those are definitely on the radar as well.  I have a plugin [here](https://github.com/tlambert03/fonticon-materialdesignicons6) that allows them to be used from the new fonticon approach i'm building in [superqt](https://github.com/napari/superqt/pull/24)
